shimmingtoolbox/cli/image.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*
+
+import click
+import os
+import nibabel as nib
+
+from shimmingtoolbox.image import concat_data
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+AXES = ['0', '1', '2', '3', '4']
+
+
+@click.group(context_settings=CONTEXT_SETTINGS,
+             help="Perform manipulations on images")
+def image_cli():
+    pass
+
+
+@image_cli.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('input', nargs=-1, type=click.Path(exists=True), required=True)
+@click.option('--output', 'fname_output', type=click.Path(), default=os.path.join(os.curdir, 'concat.nii.gz'),
```

Comment:
if users don't read the usage (users never read the doc) and fail to provide an output name, the file concat is created and they don't know about this. So either force output flag to be used (not great if ST will mostly be used with the GUI) or indicate in the terminal what will be the output.

alternatively, you could create a file name where you concatenate the file names of the input eg:
~~~
st_image concat fmri1.nii.gz fmri2.nii
--> Created: fmri1_fmri2.nii
~~~